README.md

```diff
@@ -59,7 +59,35 @@ When facilitating the workshop in an offline environment, you can run `gitbook s
 If you want to generate course assets yourself, simply run `./install-dependencies.sh` and `./build.sh`. You will find the generated assets in the `output` folder. The `./package.sh` script is used to zip up the generated assets into downloadable archives and to create the course website.
 
 [Travis CI](https://travis-ci.org/) is configured to build, package, and publish a new release to GitHub Pages and Releases whenever a new tag is pushed. So all you need to create a new release is to push a new tag with `git tag <version>` and `git push --tag`.
-  
+
+## Hardware
+
+Each node consists of one:
+
+* Raspberry Pi 3B or 3B+ with power supply
+* SD card with 8 GB or more space
+* [TOP-S5 USB WiFi adapter](https://github.com/phillymesh/802.11s-adapters/blob/master/toplinkst-top-s5/toplinkst-top-s5.md) from [Toplinkst](https://toplinkst-wifi.en.made-in-china.com/product/NyaQzvYEsGWL/China-Top-S5-802-11A-B-G-N-300Mbps-Rt5572-Dual-Band-USB-Wireless-Module-with-Ce-FCC-for-STB.html)
+
+It is possible to use other [USB WiFi adapters](https://github.com/phillymesh/802.11s-adapters/blob/master/README.md) with `rt2800usb`, `ath9k_htc`, or `rtl8192cu` drivers with [slight modifications to the software](https://github.com/benhylau/mesh-workshop/tree/master/src/custom/etc/systemd/network).
+
+Other accessories:
+
+* Ethernet cables
+* A network switch may come in handy
+* LoRa hardware for _Module 6: Living on a decentralized network_
+
+## Software
+
+1. Download the [latest mesh-orange release](https://github.com/benhylau/mesh-router-builder/releases/latest) of `raspberrypi3-<version>_default.img`, then flash the image onto an SD card with a tool like [Etcher](https://etcher.io).
```

Comment:
While etcher.io still works the current URL is `https://www.balena.io/etcher/`

Reply:
Oh didn't know that happened. They will probably keep it but still good to update. Thanks.